packages/blue-sdk/src/chain.ts

```diff
@@ -101,6 +101,28 @@ export namespace ChainUtils {
       logoSrc: "https://cdn.morpho.org/assets/chains/base.png",
       identifier: "base",
     },
+    [ChainId.ArbitrumMainnet]: {
+      name: "Arbitrum Mainnet",
+      id: ChainId.ArbitrumMainnet,
+      nativeCurrency: { name: "Ether", symbol: "ETH", decimals: 18 },
+      defaultRpcUrl: "https://arb1.arbitrum.io/rpc",
+      explorerUrl: "https://arbiscan.io",
+      isTestnet: false,
+      shortName: "Arbitrum",
+      logoSrc: "https://cdn.morpho.org/assets/chains/arbitrum.png",
```

Comment:
This logo does not exist, we should either upload it, make `logoSrc` optional OR ref another valid logo url (we probably should not as it would  not be a trusted url)